openmetadata-ui/src/main/resources/ui/src/pages/MyDataPage/MyDataPage.component.tsx

```diff
@@ -410,15 +424,31 @@ const MyDataPage = () => {
       });
   };
 
+  // Fetch tasks list to show cound for Pending tasks
+  const fetchMyTaskData = useCallback(() => {
+    if (!currentUser || !currentUser.id) {
+      return;
+    }
+
+    getFeedsWithFilter(
+      currentUser.id,
+      FeedFilter.ASSIGNED_TO,
+      undefined,
```

Comment:
Can we use null?

Reply:
Sending `undefined` as param is optional, If we replace it with `null` then it will enforce to send null in each case

Reply:
We should move optional arguments to the last and provide a default value and not use undefined - https://rules.sonarsource.com/typescript/RSPEC-4623